griptape/drivers/prompt/ai21_prompt_driver.py

```diff
@@ -0,0 +1,32 @@
+import ai21
+from attr import define, field, Factory
+from griptape.artifacts import TextArtifact
+from griptape.drivers import BasePromptDriver
+from griptape.tokenizers import Ai21Tokenizer
+
+
+@define
+class Ai21PromptDriver(BasePromptDriver):
```

Comment:
No need to set custom prefix or suffix for their models?

Reply:
As far as I can tell from the [docs](https://docs.ai21.com/reference/j2-complete-api-ref), no. Though they do recommend certain [prompting techniques](https://docs.ai21.com/docs/prompt-engineering) that we might want to play with once we've got driver-specific prompt templates.